fixieai/cli/agent/commands.py

```diff
@@ -167,14 +167,11 @@ def init_agent(handle, description, entry_point, more_info_url, requirement, lan
 
     is_typescript = language.lower() in ["typescript", "ts"]
 
-    try:
-        current_config = agent_config.load_config()
-    except FileNotFoundError:
-        current_config = agent_config.AgentConfig()
+    current_config = agent_config.AgentConfig()
     current_config.handle = handle
     current_config.description = description
     current_config.entry_point = entry_point
-    current_config.language = language
+    current_config.language = "typescript" if is_typescript else "python"
```

Comment:
If we just set `current_config.language = language`, then we could end up with it being `ts` or `typescript` or `py` or `python`. This normalizes it.

Reply:
We probably can make the `language` field an enum and/or just change its type to `Literal["typescript", "python"]` to let the type-checker verify.

Reply:
I tried to make it an enum two days ago and ran into a series of problems that made me scrap the attempt. But maybe someone better at Python can do it better. :smile: